### PR TITLE
Proof P76 (#239)

### PR DESCRIPTION
**Pull request by keller0**
Proof P76 [#239](https://github.com/chrislgarry/Apollo-11/issues/239)

### CODE REVIEW

File: Comanche055/P76.agc [hunk 1]
```diff
@@ -35,10 +35,10 @@
 #	TIME (TIG) AT WHICH THE OTHER VEHICLE VELOCITY WAS CHANGED BY EXECUTION OF A THRUSTING MANEUVER. THE
```
Comment (wopian) [inline Comanche055/P76.agc]: ✨ `--` instead of `-` on lines `32` and `33`

File: Comanche055/P76.agc [hunk 2]
```diff
@@ -35,10 +35,10 @@
...
 #	OTHER VEHICLE STATE VECTOR IS INTEGRATED TO TIG AND UPDATED BY THE ADDITION OF DELTA V (DELTA V HAVING
 #	BEEN TRANSFORMED FROM LV TO REF COSYS).  USING INTEGRVS, THE PROGRAM THEN INTEGRATES THE OTHER
-#	VEHICLE STATE VECTOR TO THE STATE VECTOR OF THIS VEHICLE, THUS INSURING THAT THE W-MATRIX AND BOTH VEHICLE
+#	VEHICLE STATE VECTOR TO THE STATE VECTOR OF THIS VEHICLE. THUS INSURING THAT THE W-MATRIX AND BOTH VEHICLE
```
Comment (wopian) [inline Comanche055/P76.agc]: ❌ This is yet another badly printed comma - you can partially see the flick through the jpeg artefacts

File: Comanche055/P76.agc [hunk 3]
```diff
@@ -35,10 +35,10 @@
...
 #	STATES CORRESPOND TO THE SAME TIME.
 # 3)	ERASABLE INITIALIZATION REQUIRED -- NONE.
-# 4)	CALLING SEQUENCES AND EXIT MODES -- CALLED BY ASTRONAUT REQUEST THRU DSKY V 37 E 76E.
+# 4)	CALLING SEQUENCES AND EXIT MODES -- CALLED BY ASTRONAUT REQUEST THRU DSKY V 37 E 76 E.
```
Comment (wopian) [inline Comanche055/P76.agc]: ✨ `--` to `-` for all the lines around this

File: Comanche055/P76.agc [hunk 4]
```diff
@@ -35,10 +35,10 @@
...
 #	EXITS BY TCF ENDOFJOB.
 # 5)	OUTPUT -- OTHER VEHICLE STATE VECTOR INTEGRATED TO TIG AND INCREMENTED BY DELTA V IN REF COSYS.
 #	THE PUSHLIST CONTAINS THE MATRIX BY WHICH THE INPUT DELTA V MUST BE POST-MULTIPLIED TO CONVERT FROM LV
```
Comment (wopian) [inline Comanche055/P76.agc]: For line `# 7)...`

✨ No spacing between list of `BANKCALL` to `CSMPREC`
✨ No spacing at `LEMPREC),ATOPCSM`
✨ No spacing between list of `ATOPLEM)` to `INTWAKE`

For line `# 8)...`

✨ No spacing between list of `MOONFLAG` to `RASFLAG`

File: Comanche055/P76.agc (hunk 1 above)
Comment (keller0) [reply Comanche055/P76.agc]: Done

**Comment by wopian** [comment]
Thanks for this PR :tada:

**Pull request merged**